packages/nextcloud/test/webdav_test.dart

```diff
@@ -201,14 +201,20 @@ void main() {
       setUpAll(() async {
         container = await DockerContainer.create(preset);
         client = await TestNextcloudClient.create(container);
+
+        await client.webdav.mkcol(PathUri.parse('webdav'));
```

Comment:
Can we just call this `test`?
Looking at the fixtures, I was really confused why the uri was `/webdav/webdav`.

If it needs to be unique, I'd rather use `webdav_test`